src/scene/immediate.js

```diff
@@ -113,6 +113,14 @@ pc.extend(pc.Application.prototype, function () {
         lineBatches[batchId].addLines(position, color);
     }
 
+    function _destroyLines() {
+        var lineVertexFormat = null;
+        var lineBatches = [];
+        var quadMesh = null;
+        var cubeLocalPos = null;
+        var cubeWorldPos = null;
```

Comment:
Putting `var` at the start of these lines makes this function do exactly nothing :)

I believe you're trying to reset the values of these variables in the outer scope where they are varred, so you can just remove `var` on lines 117-121 and that will do its thing.

Reply:
😱 I make a big mistake...